functions/src/notifications/prayerRequestPrayed.js

```diff
@@ -0,0 +1,40 @@
+const { firestore, functions, messaging, admin } = require('../index');
+
+exports.prayerRequestPrayedNotification = functions
```

Comment:
We would also need to check if the user has prayer request notifications turned on